lighthouse-core/test/test-utils.js

```diff
@@ -162,6 +162,18 @@ function loadSourceMapAndUsageFixture(name) {
   };
 }
 
+/**
+ * @param {string} name
+ * @return {{devtoolsLog: LH.DevtoolsLog, trace: LH.Trace}}
+ */
+function loadTraceFixture(name) {
+  const dir = `${LH_ROOT}/lighthouse-core/test/fixtures/traces`;
+  return {
+    devtoolsLog: JSON.parse(fs.readFileSync(`${dir}/${name}.devtools.log.json`, 'utf-8')),
+    trace: JSON.parse(fs.readFileSync(`${dir}/${name}.json`, 'utf-8')),
```

Comment:
Not every trace follows this naming scheme. Maybe make `loadTraceFixture` and `loadDevtoolsLogFixture` and use the entire filename?

Reply:
Deferring on this, but I think we can just rename the other files. Splitting makes sense if only one of these two exists, but will wait for when other tests use this file (which isn't necessary unless they are in the tsconfig)